### PR TITLE
SHARE-555 child restriction

### PR DESCRIPTION
**Pull request by makne17**
added not for kids property to admin panel
added button to change not for kids flag
added not for kids flag in meta data

---

### Your checklist for this pull request

Please review the [contributing guidelines](https://github.com/Catrobat/Catroweb-Symfony/blob/develop/.github/contributing.md) and [wiki pages](https://github.com/Catrobat/catroweb-Symfony/wiki/) of this repository.

- [ ] Include the name and id of the Jira ticket in the PR’s title eg.: `SHARE-666 The devils ticket`
- [ ] Choose the proper base branch (_develop_)
- [ ] Confirm that the changes follow the project’s coding guidelines
- [ ] Verify that the changes generate no warnings and errors
- [ ] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Verify that all tests are passing (CI), if not please state the test cases in the [section](#Tests) below
- [ ] Perform a self-review of the changes
- [ ] Stick to the project’s git workflow (rebase and squash your commits)
- [ ] Verify that your changes do not have any conflicts with the base branch
- [ ] Put your ticket into the `Code Review` section in [Jira](https://jira.catrob.at/)
- [ ] Post a message in the _#catroweb_ [Slack channel](https://catrobat.slack.com) and ask for a code reviewer
- [ ] Check that your pull request has been successfully deployed to https://web-test-1.catrob.at/

### Additional Description

`TODO: Add additional information that is not in your commit-message here`

### Tests - additional information

`TODO: add additional information about testruns here`

### CODE REVIEW

**Comment by makne17** [comment]
CAPI Model/ProjectResponse fehlt noch die Variable not_for_kids, hab sie zur catroweb.yaml file hinzugefügt wurde auch schon gemerged 

Edit: -> https://github.com/Catrobat/Catroweb/pull/4799

File: translations/catroweb.en.yaml [hunk 1]
```diff
@@ -1038,5 +1042,6 @@ maintenanceinformations:
     feature_2: 'The Feature'
     feature_3: 'The Feature'
 
-
+MarkNotForKids: Mark as not for kids
```
Comment (dmetzner) [inline translations/catroweb.en.yaml]:
```suggestion
MarkNotForKids: Mark as not safe for kids
```

File: translations/catroweb.en.yaml [hunk 2]
```diff
@@ -1027,6 +1028,9 @@ media_library:
 
 snackbar:
   project_not_found: 'The requested project is not available'
+  project_not_for_kids: 'The project was marked as not for kids!'
```
Comment (dmetzner) [inline translations/catroweb.en.yaml]:
```suggestion
  project_not_for_kids: 'The project was marked as safe not for kids!'
```

File: src/Application/Controller/Project/ProjectController.php [hunk 3]
```diff
@@ -418,6 +418,29 @@ public function projectCommentDetail(string $id): Response
...
   }
 
+  #[Route(path: '/markNotForKids/{id}', name: 'mark_not_for_kids', methods: ['POST'])]
+  public function markNotForKids(string $id): Response
+  {
+    $project = $this->project_manager->find($id);
+    if (null === $project) {
+      return $this->redirectToIndexOnError();
+    }
+    if (2 == $project->getNotForKids()) {
```
Comment (dmetzner) [inline src/Application/Controller/Project/ProjectController.php]:
```suggestion
    if (<const>== $project->getNotForKids()) {
```

Use constants please

**Comment by dmetzner** [comment]
@makne17 api is now merged, just rebase :)

**Review by dmetzner — APPROVED** [review]

**Pull request merged**